indico/MaKaC/webinterface/rh/categoryDisplay.py

```diff
@@ -61,6 +61,9 @@ class RHCategDisplayBase(base.RHDisplayBaseProtected):
     def _checkParams(self, params, mustExist=True):
         if "categId" in params:
             params["categId"] = escape_html(str(params["categId"]))
+        # TODO: Workaround to avoid breaking the category navigation (display)
```

Comment:
This RH will go away when we move category display to Jinja and new RHs so I wouldn't worry too much.